test/release/examples/primers/Makefile

```diff
@@ -38,37 +38,41 @@ TARGETS = \
 	varargs \
 	variables \
 
+CLEAN_TARGETS=$(TARGETS)
+
 #
 # Only make FFTW if FFTW_DIR is set or we can find the fftw library
 #
 ifdef FFTW_DIR
 	TARGETS += FFTWlib
 	FFTW_OPTS = -I$(FFTW_DIR)/include -L$(FFTW_DIR)/lib
 endif
+CLEAN_TARGETS += FFTWlib
 
 ifdef CRAY_LIBSCI_PREFIX_DIR
 	TARGETS += LAPACKlib LinearAlgebralib
 	LAPACK_OPTS = -lgfortran -lsci_gnu
 endif
+CLEAN_TARGETS += LAPACKlib LinearAlgebralib
 
 #
-# Only make interopWithC and cClient if LLVM backend is set
+# Only make interopWithC if LLVM backend is set
 #
 CHPL_TARGET_COMPILER=$(shell $(CHPL_HOME)/util/chplenv/chpl_compiler.py --target)
 ifeq ($(CHPL_TARGET_COMPILER),llvm)
-	TARGETS += cClient
 	TARGETS += interopWithC
 endif
-
+CLEAN_TARGETS += interopWithC
```

Comment:
Though it doesn't have a `_real` counterpart, I think `cClient` still needs to be added to CLEAN_TARGETS.  This could either be done after CLEAN_REALS is set up, or it could just be added here (which is probably what I'd do, though I don't feel strongly either way).